lib/ZwaveClient.js

```diff
@@ -2376,45 +2376,51 @@ ZwaveClient.prototype.hardReset = async function () {
 
   throw Error('Driver is closed')
 }
-
-ZwaveClient.prototype.sendCommand = async function (valueId, command, args) {
+/**
+ * Send a command
+ *
+ * @param {{nodeId: number, endpoint: number, commandClass: number}} ctx context to get the instance to send the command
+ * @param {string} command command name
+ * @param {any[]} args args to pass to `command`
+ * @returns {Promise<any>}
+ */
+ZwaveClient.prototype.sendCommand = async function (ctx, command, args) {
   if (this.driver && !this.closed) {
-    if (typeof valueId.nodeId !== 'number') {
+    if (typeof ctx.nodeId !== 'number') {
       throw Error('nodeId must be a number')
     }
 
-    const error = utils.isValueId(valueId)
-
-    if (typeof error === 'string') {
-      throw Error(error)
-    }
-
     if (args !== undefined && !Array.isArray(args)) {
       throw Error('if args is given, it must be an array')
     }
 
-    const node = this.getNode(valueId.nodeId)
+    // get node instance
+    const node = this.getNode(ctx.nodeId)
     if (!node) {
-      throw Error(`Node ${valueId.nodeId} was not found!`)
+      throw Error(`Node ${ctx.nodeId} was not found!`)
     }
-    const endpoint = node.getEndpoint(valueId.endpoint || 0)
+
+    // get the endpoint instance
+    const endpoint = node.getEndpoint(ctx.endpoint || 0)
     if (!endpoint) {
       throw Error(
-        `Endpoint ${valueId.endpoint} does not exist on Node ${valueId.nodeId}!`
+        `Endpoint ${ctx.endpoint} does not exist on Node ${ctx.nodeId}!`
       )
     }
-    const api = endpoint.commandClasses[valueId.commandClass]
 
+    // get the command class instance to send the command
+    const api = endpoint.commandClasses[ctx.commandClass]
     if (!api || !api.isSupported()) {
       throw Error(
-        `Node ${valueId.nodeId} (Endpoint ${valueId.endpoint}) does not support CC ${valueId.commandClass}`
+        `Node ${ctx.nodeId} (Endpoint ${ctx.endpoint}) does not support CC ${ctx.commandClass}`
```

Comment:
The message is not 100% correct. If `!api` then the CC API is not implemented. That does **not** mean that the endpoint does not support hte CC.

Reply:
I think that error is the same I copied from your io.broker moudule. BTW I have make it more clear now